src/patterns/components/footer/react/code/footer.hbs

```diff
@@ -0,0 +1,212 @@
+        analyticsString: 'about-this-link',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'About This Other Thing',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'About That',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'Link Item',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'This Link Item',
+        linkElement: 'a',
+      },
+    ],
+  },
+  {
+    heading: 'Learn More',
+    links: [
+      {
+        href: '#nogo',
+        text: 'About This Other Thing',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'About This',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'About That',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'Link Item',
+        linkElement: 'a',
+      },
+    ],
+  },
+  {
+    heading: 'Support',
+    links: [
+      {
+        href: '#nogo',
+        text: 'Share Your Screen',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'Opt Out',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'Disclosures and Other Things',
+        linkElement: 'a',
+      },
+      {
+        href: '#nogo',
+        text: 'We Want Your Feedback',
+        linkElement: 'a',
+      },
+    ],
+  },
+];
+
+const connectIcons = {
+  heading: 'Connect With Us',
+  icons: [
+    {
+      href: '#nogo',
+      name: 'facebook',
+      screenReaderText: 'Facebook',
+      linkElement: 'a',
+    },
+    {
+      href: '#nogo',
+      name: 'instagram',
+      screenReaderText: 'Instagram',
+      linkElement: 'a',
+    },
+    {
+      href: '#nogo',
+      name: 'twitter',
+      screenReaderText: 'Twitter',
+      linkElement: 'a',
+    },
+    {
+      href: '#nogo',
+      name: 'youtube',
+      screenReaderText: 'Youtube',
+      linkElement: 'a',
+    },
+  ],
+};
+
+const awards = {
+  heading: 'Awards Heading Title',
+  images: [
+    {
+      href: '#nogo',
+      src: 'https://sparkdesignsystem.com/assets/toolkit/images/spark-logo.svg',
+      altText: 'Spark Logo',
+      addClasses: 'drizzle-c-Logo',
+      linkElement: 'a',
+    },
+    {
+      href: '#nogo',
+      src: 'https://sparkdesignsystem.com/assets/toolkit/images/spark-logo.svg',
+      altText: 'Spark Logo',
+      addClasses: 'drizzle-c-Logo',
+      linkElement: 'a',
+    },
+  ],
+  disclaimerTitle: 'My Award Disclaimer',
+  disclaimerAnalytics: 'disclaimer-1',
+  disclaimerText: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam in laoreet ante, non feugiat neque. Suspendisse et ipsum leo. Quisque non consectetur justo.',
+};
+
+const paragraphs = [
+  { text: '*Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam' },
```

Comment:
it looks like this could just be an array containing text

Reply:
I went this route to allow for each paragraph to have additional keys like the generated ID.